build/README.md

```diff
@@ -43,6 +44,17 @@ There are 3 different containers instances that are run from this image.  The fi
 
 All Docker names are suffixed with a hash derived from the file path (to allow concurrent usage on things like CI machines) and a version number.  When the version number changes all state is cleared and clean build is started.  This allows the build infrastructure to be changed and signal to CI systems that old artifacts need to be deleted.
 
+## Minikube
+
+To use minikube you need to create an SSH tunnel for rsync, do something like this:
+
+```
+export KUBE_RSYNC_PORT=8730
+eval $(minikube docker-env)
+ssh -i ~/.minikube/machines/minikube/id_rsa  $(minikube ip) -L ${KUBE_RSYNC_PORT}:localhost:${KUBE_RSYNC_PORT} -N -l docker &
```

Comment:
ssh into  minikube with minikube creds, start a tunnel into out of the vm with the rsync port, don't do anything, and login as docker. background to keep the tunnel open and let me have my shell back.

I don't think we should advocate allocating sockets in the background so people forget about them. At least not without telling them how to clean up.